addons/l10n_in/models/account_invoice.py

```diff
@@ -21,7 +21,8 @@ class AccountMove(models.Model):
             ('deemed_export', 'Deemed Export'),
             ('uin_holders', 'UIN Holders'),
         ], string="GST Treatment", compute="_compute_l10n_in_gst_treatment", store=True, readonly=False, copy=True, precompute=True)
-    l10n_in_state_id = fields.Many2one('res.country.state', string="Place of supply", compute="_compute_l10n_in_state_id", store=True, readonly=False)
+    l10n_in_state_id = fields.Many2one('res.country.state', string="Place of supply",
```

Comment:
Should not l10n_in_state_id have copy False as well?

Reply:
Not sure...  It can work as well without as if the partner changes, it should change.